tests/test_dynamodb.py

```diff
@@ -30,17 +30,10 @@ class SimpleKeyModel(BaseModel):
     expires: datetime
     sigfig: Decimal
     enabled: bool
-    data: Dict[int, int] = None
+    data: Dict[int, int] = {}
```

Comment:
```suggestion
    data: Dict[int, int]
```

The previous None was there to "document" it as optional (poorly). I think we can leave off the `{}` because it's out-of-step with everything around it.